packages/puppeteer/src/node/install.ts

```diff
@@ -48,24 +48,64 @@ export async function downloadBrowser(): Promise<void> {
 
   const unresolvedBuildId =
     configuration.browserRevision || PUPPETEER_REVISIONS[product] || 'latest';
+  const unresolvedShellBuildId =
+    configuration.browserRevision ||
+    PUPPETEER_REVISIONS['chrome-headless-shell'] ||
+    'latest';
+
+  const shellBuildId =
```

Comment:
We should move it just before it is used else it may fail while you want only Chrome without shell.

Reply:
Updated error reporting